### DISCUSSION TITLE
DisplayCal on Windows

### DISCUSSION THREAD

**Discussion by ziemaf** [eoyilmaz/displaycal-py3#183] [Q&A] [answered]
Hi everyone 
I am glad that the DisplayCal project has a chance for a second life.
Is there a chance for a Windows version?

**Comment by p5k369** [comment, marked as answer]
Hey @ziemaf, note that there is already an issue #34. 
The thing is, DisplayCal will start on windows10 however there are some issues I think with the handling of subprocesses which result in calibration and profiling not working atm. 
These issues are windows specific and propably have to do with the main and the wexpect module.
I think it is very likely that there will be a windows version at some point if that is the only problem that has to be addressed.

**Comment by BiscuitCookies** [comment]
Any news/progress on a possible windows version? Or is it still a ways of?
EDIT: Windows installer is now added to releases
